subprojects/settings.gradle.kts

```diff
@@ -1,5 +1,76 @@
 enableFeaturePreview("VERSION_CATALOGS")
-enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
+
+pluginManagement {
+    includeBuild("../build-logic-settings")
+
+    val artifactoryUrl: String? by settings
+
+    repositories {
+        maven {
```

Comment:
Could we make an extension function `mavenWithProxy` that will have 3 parameters: title, URL, proxy URL
To reduce duplications

Reply:
looks like not in pluginManagement